pkgs/data/fonts/line-awesome/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchurl, unzip }:
+
+let
+  version = "1.3.0";
```

Comment:
You can use the normal `version` and `pname` split within `stdenv.mkDerivation` now.